Autocomplete/predefined-items-and-recent-searches/src/predefined-items.js

```diff
@@ -0,0 +1,51 @@
+export const predefinedItems = {
+  getSources({ query }) {
+    if (query) {
+      return [];
+    }
+
+    return [
+      {
+        getSuggestions() {
+          return [
+            {
+              title: 'GitHub repository',
+              url: 'https://github.com/algolia/autocomplete.js/tree/next',
+            },
+            {
+              title: 'Documentation website',
+              url: 'https://algolia-autocomplete.netlify.app',
+            },
+            {
+              title: 'Algolia',
+              url: 'https://www.algolia.com',
+            },
+          ];
+        },
+        getSuggestionUrl(suggestion) {
+          return suggestion.url;
+        },
+        templates: {
+          header() {
+            return `
+              <div class="aa-PredefinedHeader">
+                <p>Useful Links</p>
+              </div>
+            `;
+          },
+          item({ item, root }) {
+            return `
+              <div class="aa-PredefinedItem">
```

Comment:
We should make the whole container a link (wrapping the `div` with `a`).

Reply:
Can you tell me why?

Reply:
Otherwise if you click anywhere after the "GitHub repository" text, the dropdown closes without redirecting to the link.

![image](https://user-images.githubusercontent.com/6137112/97162304-52a89f00-177f-11eb-89fb-51dbd24c17e8.png)